logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/GetPublicAssetUseCase.kt

```diff
@@ -0,0 +1,16 @@
+package com.wire.kalium.logic.feature.asset
+
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.data.asset.AssetRepository
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.functional.suspending
+
+interface GetPublicAssetUseCase {
+    suspend operator fun invoke(assetKey: String): Either<CoreFailure, ByteArray>
```

Comment:
Can't we have `assetKey` be of type `AssetId` or `UserAssetId`?

Reply:
Thanks, changes applied, completely agree that using this parameter suits it better